samples/Monitoring/deploy-mmaextension-windows-tags-loganalytics/azurepolicy.json

```diff
@@ -0,0 +1,140 @@
+{
+    "properties": {
+        "displayName": "Deploy Microsoft Monitoring Agent on Windows Machines based on a Tag",
+        "mode": "Indexed",
+        "description": "Deploy Microsoft Monitoring Agent on Windows Machines based on a Tag with value: MMA and Name: Windows",
+        "metadata": {
+            "category": "Batch"
+        },
+        "parameters": {
+            "logAnalytics": {
+                "type": "String",
+                "metadata": {
+                    "displayName": "Log Analytics workspace",
+                    "description": "Select Log Analytics workspace from dropdown list. If this workspace is outside of the scope of the assignment you must manually grant 'Log Analytics Contributor' permissions (or similar) to the policy assignment's principal ID.",
+                    "strongType": "omsWorkspace",
+                    "assignPermissions": true
+                }
+            },
+            "tagName": {
+                "type": "String",
+                "metadata": {
+                    "displayName": "Tag Name",
+                    "description": "Name of the tag, such as 'production'"
+                }
+            },
+            "WorkspaceID": {
+                "Type": "String",
+                "metadata": {
+                    "displayName": "Log Analytics workspace",
+                    "description": "Name of the workspace, such as 'production'"
+                }
```

Comment:
This isn't accurate.  The WorkspaceId needs to be the .customerId from the workspace in order for this policy to work... Why not get it via reference() like the built-in policy?